### PR TITLE
[Linux] Crash when remote debugging with Web UI

### PR DESCRIPTION
**Pull request by PeterWangIntel**
The Web UI doesn't have favicon, so don't need to show it in front-end.

And the reason of crash is that the delegate of 'WebContents' related with 'File Picker' is 'WeDialogView' rather than 'Runtime', so 'static_cast&lt;Runtime*&gt;' in 'GetFaviconDataURL' will cause problem.

With this patch the 'File Picker' can be debugged, like attached figure shows.
remote-debug-web-ui

BUG=https://crosswalk-project.org/jira/browse/XWALK-4185

### CODE REVIEW

**Comment by PeterWangIntel** [comment]
@pozdnyakov @halton @joone , please help to review, thank you.

**Comment by rakuco** [comment]
"Crush" should be "Crash".

File: runtime/browser/devtools/xwalk_devtools_delegate.cc [hunk 1]
```diff
@@ -82,25 +84,23 @@ class Target : public content::DevToolsTarget {
 
 Target::Target(scoped_refptr<content::DevToolsAgentHost> agent_host)
     : agent_host_(agent_host) {
+  favicon_url_ = GURL();
```
Comment (pozdnyakov) [inline runtime/browser/devtools/xwalk_devtools_delegate.cc]: I believe this is not needed, favicon_url_ is already constructed as GURL()

Comment (PeterWangIntel) [inline runtime/browser/devtools/xwalk_devtools_delegate.cc]: yes, sorry.

**Comment by pozdnyakov** [comment]
lgtm

**Comment by rakuco** [comment]
Please backport if necessary.

**Pull request merged**